### PR TITLE
Use dlvm TensorFlow whl

### PR DESCRIPTION
**Pull request by rosbo**
Use the new wheel built in #772

### CODE REVIEW

**Comment by rosbo** [comment]
I canceled the `whl` build which was triggered because this branch name has the `-whl` suffix. This is why it says the checks have failed.

However, the main build is still in progress and hasn't failed yet: https://ci.kaggle.net/job/kernels/job/docker-python/job/use-dlvm-tf-whl/

File: gpu.Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ARG BASE_TAG=staging
 
 FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04 AS nvidia
-FROM gcr.io/kaggle-images/python-tensorflow-whl:2.1.0-py37 as tensorflow_whl
+FROM gcr.io/kaggle-images/python-tensorflow-whl:2.1.0-py37-2 as tensorflow_whl
```
Comment (vimota) [inline gpu.Dockerfile:4]: Where does the name "py37-2" come from? I only saw it in the other PR branch in the change logs, but I'm guessing that naming is set via a convention somewhere?

Comment (rosbo) [reply gpu.Dockerfile:4]: The convention is `{TF_VERSION}-py{PYTHON-VERSION}{optional version if multiple images are built for the same tf and python version}`

**Comment by rosbo** [comment]
The tag to `gcr.io/kaggle-images/python-tensorflow-whl` is added manually. This is why this build can pass even if #772 hasn't been merged. I don't want to merge #772 until I ensure it is working for the main build.

**Comment by vimota** [comment]
Got it thanks!!

**Comment by rosbo** [comment]
Failed only the jax test which has been fixed in #771 .

**Pull request merged**